### PR TITLE
Adjust snapshot padding/margin

### PR DESCRIPTION
**Pull request by mparke**
before:
screen shot 2017-02-14 at 1 30 42 pm

after:
screen shot 2017-02-14 at 1 43 08 pm

### CODE REVIEW

**Comment by mparke** [comment]
@eatonphil fixed, see screencap

**Comment by eatonphil** [comment]
Sorry to keep nitpicking, but could you vertically center the entire thing? (Not just the not-button-text.)

**Comment by mparke** [comment]
screen shot 2017-02-15 at 3 57 26 pm

File: src/linodes/linode/backups/layouts/SummaryPage.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class SummaryPage extends Component {
 
   renderEmptySnapshot() {
     return (
-      <div className="Backup col-sm-3" key="Snapshot">
+      <div className="Backup Backup-empty-snapshot col-sm-3" key="Snapshot">
```
Comment (eatonphil) [inline src/linodes/linode/backups/layouts/SummaryPage.js]: emptySnapshot?

**Pull request merged**